### ISSUE TITLE
Question: I noticed the nomyth option

### ISSUE THREAD

**Issue by bonelifer** [SchedulesDirect/mfdb-json#14]
Does that mean that it can also produce a RAW XMLTV file? If so how?

**Comment by rkulagowski** [comment]
No, no plans for XML output right now. The nomyth option is for developers
that want to download the JSON and insert it into their own database.

**Comment by bonelifer** [comment]
Ok, I'm just looking around. I have a Raspberry Pi 2 with OSMC on it. Setting it up to use a Kodi tv guide called FTV guide for LiveTV.  So it looks like either the XMLTV JSON client ( https://github.com/kgroeneveld/tv_grab_sd_json ) or sd2xmltv ( https://github.com/astrilchuk/sd2xmltv ). Was just hoping to use one client. Thanks for the quick reply.
